### PR TITLE
Events

### PR DESCRIPTION
**Pull request by themonster2015**
In this PR, I implemented the following:

- Copied the files calculate.js and operate.js into a logic/ directory.

- Imported the files in the Calculator component.

- Implemented the event handlers to use the math logic from calculate.js and operate.js.

- The app works well in the browser.
- Fixed divide by 0 error.

### CODE REVIEW

File: src/components/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === ':') {
```
Comment (Davidosky007) [inline src/components/logic/operate.js]: * When i do division by zero operation i get errors in the console kindly fix this 🙏 
image

Comment (Davidosky007) [inline src/components/logic/operate.js]: * Kindly change the division symbol and also fix the display so that the correct number entered shows 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @themonster2015 👋, 

Good job so far! 💪 
* Good commit messges ✔️ 
* Nice looking PR ✔️ 
* No linter errors ✔️ 
* Professional repo ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/logic/operate.js (hunk 1 above)
Comment (themonster2015) [reply src/components/logic/operate.js]: I've fixed it. Thanks

Comment (themonster2015) [reply src/components/logic/operate.js]: Thanks for pointing it out. I've fixed it.

**Review by AshakaE — APPROVED** [review]
### Hi @themonster2015  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**